qiskit/test/mock.py

```diff
@@ -233,26 +233,31 @@ def __init__(self):
             pulse_library=[PulseLibraryItem(name='test_pulse_1', samples=[0.j, 0.1j]),
                            PulseLibraryItem(name='test_pulse_2', samples=[0.j, 0.1j, 1j]),
                            PulseLibraryItem(name='test_pulse_3',
-                                            samples=[0.j, 0.1j, 1j, 0.5 + 0j])],
+                                            samples=[0.j, 0.1j, 1j, 0.5 + 0j]),
+                           PulseLibraryItem(name='test_pulse_4',
+                                            samples=7*[0.j, 0.1j, 1j, 0.5 + 0j])],
             cmd_def=[Command(name='u1', qubits=[0],
                              sequence=[PulseQobjInstruction(name='fc', ch='d0',
                                                             t0=0, phase='-P1*np.pi')]),
                      Command(name='u3', qubits=[0],
                              sequence=[PulseQobjInstruction(name='test_pulse_1', ch='d0', t0=0)]),
                      Command(name='u3', qubits=[1],
                              sequence=[PulseQobjInstruction(name='test_pulse_3', ch='d1', t0=0)]),
+                     Command(name='x', qubits=[0],
```

Comment:
I think we should avoid using `x` as it is not in the basis gates. This should be removed from the `cmd_def` returned by the backends.

Reply:
The client devices have `x` gates

Reply:
They will eventually be removed.

Reply:
Related question, how do I translate _X_ _H_, and other QASM supported gates if they don't have a command definition?

Reply:
you just error. those should have been handled by the transpiler at an earlier stage, and unrolled to the right basis gates.

Reply:
thank you!